deploy_production_mcp.py

```
@@ -181,105 +193,105 @@ async def demonstrate_unique_capabilities(self):
-                'user_data': 'personal info',
-                'public_data': 'can be shared'
+                "api_key": "secret123",
+                "user_data": "personal info",
+                "public_data": "can be shared",
             }
-            
+
             filtered = self.state_fabric._apply_privacy_filters(sensitive_context)
-            assert 'api_key' not in filtered
-            assert '<encrypted>' in filtered.get('user_data', '')
+            assert "api_key" not in filtered
```

Ruff (0.11.9)

Use of `assert` detected

(S101)